interface/FastTemplate.h

```diff
@@ -119,6 +119,9 @@ class FastHisto : public FastTemplate {
 
         T GetMax() const ;
 
+        const T & GetEdge(unsigned int i) const { return binEdges_[i]; }
```

Comment:
Useful to have direct access to these for horizontal morphing and rebinning purposes. Shouldn't have any negative impact.